Sources/Components/StatisticsView/StatisticsItemEmptyView.swift

```diff
@@ -8,6 +8,12 @@ public class StatisticsItemEmptyView: UIView {
 
     // MARK: - Private
 
+    private lazy var hairlineView: UIView = {
+        let label = UILabel(withAutoLayout: true)
```

Comment:
Ikke for å være vanskelig altså, men kanskje ikke UILabel her heller?

Reply:
Fixed